### PR TITLE
Add installer compatibility to composer.json

### PR DESCRIPTION
**Pull request by ryanmarkel**
Allows people to utilize `composer` to add, install, and update the plugin from the source repo.

### CODE REVIEW

**Comment by bcampeau** [comment]
Can you check why this failed unit tests?

**Comment by ryanmarkel** [comment]
Cycling this PR to make the checks run again.

**Comment by ryanmarkel** [comment]
@bcampeau It passed this time :)

File: composer.json [hunk 1]
```diff
@@ -1,4 +1,12 @@
 {
+    "name"       : "alleyinteractive/apple-news",
+    "description": "Apple News delivery for WordPress.",
```
Comment (bcampeau) [inline composer.json]: Please change to "Enable your WordPress blog content to be published to your Apple News channel."

**Comment by bcampeau** [comment]
Totally forgot about this, my fault!

**Pull request merged**